react-native-in-app-payments-quickstart/android/app/src/debug/AndroidManifest.xml

```diff
@@ -4,7 +4,5 @@
 
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
 
-    <application android:usesCleartextTraffic="true" tools:targetApi="28" tools:ignore="GoogleAppIndexingWarning">
-        <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
-    </application>
+    <application android:usesCleartextTraffic="true" tools:targetApi="28" tools:ignore="GoogleAppIndexingWarning"/>
```

Comment:
Our direct IAP test app does _not_ have `usesCleartextTraffic`... do we know what in the plugin quickstart uses raw `http://` not `https://`?